workers/task-driver/Cargo.toml

```diff
@@ -61,13 +61,17 @@ alloy-primitives = { workspace = true }
 
 clap = { version = "4.0", features = ["derive"] }
 colored = "2"
+config = { workspace = true }
+darkpool-client = { workspace = true, features = ["arbitrum"] }
```

Comment:
we probably can't have the `arbitrum` feature set here, will lead to issues when building with the `base` feature set right